USFMToolsSharp/Models/Markers/WMarker.cs

```diff
@@ -43,7 +43,7 @@ public override string PreProcess(string input)
 
             }
 
-            return string.Empty;

```

Comment:
We would also need to add TextBlock to allowed markers in order for this to work as expected, otherwise we have a WMarker with the word set and then a text block following it, we would want this to contain that text block. Also if we go down this route I don't think we want to include an attribute for term here, there are others that we do this with we would want to handle those also.